GCTC-NTGC/gc-digital-talent
# Display results of ApplicantFilter in SingleSearchRequestTable

Resolves #3648.

This should be the minimal viable solution to #3648, ie allowing admins to review the list of applicants who meet the requirements of an ApplicantFilter request. We can spin off any non-essential features of #3648 into other tickets.

Testing:
1. ensure ApplicantSearch feature flag is true
2. Go to /search, and submit a request that results in at least some candidates.
3. Sign in to the admin dashboard. View the most recent request.
4. The table of users at the bottom should have the same number of rows as the number of candidates according to the search page.
5. Each row should have a link to the Users page, with the tabs
6. The filters in the table should be editable (without modifying the original request!), but if they're not, make that a new issue.

- [x] e567fbe rename `transform*` functions https://github.com/GCTC-NTGC/gc-digital-talent/pull/4623#issuecomment-1307431675
- [x] link ticket to improve seeders  from https://github.com/GCTC-NTGC/gc-digital-talent/pull/4623#issuecomment-1307456587
  - https://github.com/GCTC-NTGC/gc-digital-talent/issues/4357#issuecomment-1307639764
- [x] create ticket for cypress tests from https://github.com/GCTC-NTGC/gc-digital-talent/pull/4623#issuecomment-1307456587
  - https://github.com/GCTC-NTGC/gc-digital-talent/issues/4666
